modules/mod_acl_user_groups/mod_acl_user_groups.erl

```diff
@@ -601,19 +601,22 @@ manage_datamodel(Context) ->
 
             predicates=
                 [
-                    {hasusergroup, 
+                    {hasusergroup,
                         [{title, {trans, [{en, <<"In User Group">>},{nl, <<"In gebruikersgroep">>}]}}],
                         [{person, acl_user_group}]
                     },
-                    {hascollabmember, 
+                    {hascollabmember,
                         [{title, {trans, [{en, <<"Member">>},{nl, <<"Lid">>}]}}],
                         [{acl_collaboration_group, person}]
                     },
-                    {hascollabmanager, 
+                    {hascollabmanager,
                         [{title, {trans, [{en, <<"Manager">>},{nl, <<"Beheerder">>}]}}],
                         [{acl_collaboration_group, person}]
                     }
-                ]
+                ],
+            data = [
+                {acl_rules, acl_default_rules:get_default_rules()}
+            ]
```

Comment:
Correct. That’s why I’m keeping the set of default rules to the bare minimum, hopefully covering the 80%. Do you nonetheless see some problems with it?

Of course, if you really do not want these default rules, you can always `observe_manage_data(#manage_data{module = Module, props = {acl_rules, Rules}}, Context)` in your site and get rid of them or change them.